tests/modules/highway_test.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=no-self-use,invalid-name
+from numpy.testing import assert_almost_equal
+import torch
+from torch.autograd import Variable
+
+from allennlp.modules import Highway
+from allennlp.testing import AllenNlpTestCase
+
+
+class TestHighway(AllenNlpTestCase):
+    def test_forward_works_on_simple_input(self):
+        highway = Highway(2, 2)
+        # pylint: disable=protected-access
+        highway._layers[0].weight.data.fill_(1)
```

Comment:
Did you not use allennlp initialisers here for some particular reason? Maybe we need to rethink them.